base/linalg/lapack.jl

```diff
@@ -657,6 +667,7 @@ for (gesvx, elty) in
         function gesvx!(fact::Char, trans::Char, A::StridedMatrix{$elty},
                AF::StridedMatrix{$elty}, ipiv::Vector{BlasInt}, equed::Char,
                R::Vector{$elty}, C::Vector{$elty}, B::StridedVecOrMat{$elty})
+            @chktrans
             lda, n    = size(A)
```

Comment:
Ah! I hadn't checked those lines with a `SubArray`. I will update them and resubmit the PR. Thanks for the catch!